packages/telemetry/src/events/session.ts

```diff
@@ -89,18 +103,24 @@ export function eventCliSession(
 		viteVersion: getViteVersion(),
 		nodeVersion: process.version.replace(/^v?/, ''),
 		// Config Values
-		config: astroConfig
+		config: userConfig
```

Comment:
nit: this ternary got me worried for a second that we were sending the whole userConfig to telemetry. Mind moving this ternary out of the `config: ...` assignment and to the line above?